src/integration/types.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  DiscountVouchersEffectTypes,
+  StackableRedeemableResponse,
+} from '@voucherify/sdk';
+import { ChannelReference } from '@commercetools/platform-sdk/dist/declarations/src/generated/models/channel';
```

Comment:
we should not have a direct  reference into commercetools in integration namespace